relay-profiling/src/lib.rs

```diff
@@ -75,10 +80,55 @@ struct MinimalProfile {
     #[serde(alias = "profile_id", alias = "chunk_id")]
     event_id: ProfileId,
     platform: String,
+    release: Option<String>,
     #[serde(default)]
     version: sample::Version,
 }
 
+impl Filterable for MinimalProfile {
```

Comment:
Not directly related to this PR, but it might be worth defaulting these methods in the trait.